packages/viz/src/lib/sources/DOSource.ts

```diff
@@ -126,6 +129,13 @@ export class DOSource extends Source {
     this._variable = variable;
 
     this._baseURL = `${this._credentials.serverURL}api/v4/data/observatory`;
```

Comment:
Shouldn't we always use the CDN URL instead of this one?

Reply:
Nope, the CDN is only for viz endpoints not to get the metadata